internal/subshell/backend_runner.go

```diff
@@ -52,7 +52,6 @@ func (self BackendRunner) execute(executable string, args ...string) (string, er
 	}
 	env := subProcess.Environ()
 	env = append(env, "LC_ALL=C")
```

Comment:
I _think_ you can also remove this line, which sets the language to us_en for backend commands. This line was added to fix bugs like https://github.com/git-town/git-town/issues/2469.

A simple reproduction of the bug is in the ticket: `LANG=fr_FR git status`

Reply:
I'm planning to remove this in a future PR.